tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -151,22 +157,26 @@ def test_positive_deploy_configure_by_script(self):
 
         :CaseImportance: High
         """
-        name = gen_string('alpha')
-        args = self._make_virtwho_configure()
-        args.update({'name': name})
-        vhd = entities.VirtWhoConfig(**args).create()
-        self.assertEqual(vhd.status, 'unknown')
-        script = vhd.deploy_script()
+        assert virtwho_config.status == 'unknown'
+        script = virtwho_config.deploy_script()
         hypervisor_name, guest_name = deploy_configure_by_script(
             script['virt_who_config_script'], debug=True
         )
-        self.assertEqual(
-            entities.VirtWhoConfig().search(query={'search': 'name={}'.format(name)})[0].status,
-            'ok',
+        assert (
+            entities.VirtWhoConfig()
+            .search(query={'search': 'name={}'.format(virtwho_config.name)})[0]
+            .status
+            == 'ok'
         )
         hosts = [
-            (hypervisor_name, 'product_id={} and type=NORMAL'.format(self.vdc_physical)),
-            (guest_name, 'product_id={} and type=STACK_DERIVED'.format(self.vdc_physical)),
+            (
+                hypervisor_name,
+                'product_id={} and type=NORMAL'.format(settings.virtwho.sku_vdc_physical),
```

Comment:
I don't know about your Jenkins, but ours should be ready to consume Python3-compatible code. You should be able to use f-string instead of `.format()` here:
```suggestion
                f'product_id={settings.virtwho.sku_vdc_physical} and type=NORMAL',
```

Same applies to all other `.format()`s in PR.

Reply:
Changed to f-string :)